debug-on-kubernetes.md

```diff
@@ -0,0 +1,59 @@
+# Debug support on Kubernetes cluster
+
+## 1. Supported languages
+   * `java` (Required: [Debugger for Java](https://marketplace.visualstudio.com/items?itemName=vscjava.vscode-java-debug) extension)
+   * `node`
+
+## 2. Commands for debugging
+   * `Kubernetes: Debug (Launch)` - Run the current application as a Kubernetes Deployment and attach a debugging session to it (currently works only for Java/Node.js deployments)
+   * `Kubernetes: Debug (Attach)` - Attach a debugging session to an existing Kubernetes Deployment (currently works only for Java deployments)
+
+## 3. How to use it for Java debugging
+### 3.1 Launch a Spring Boot application on Kubernetes and debug it
+   * Launch VS Code.
+   * Open a Spring Boot application.
+   * If your application doesn't already have a Dockerfile, create one for it.
+   * Install [Debugger for Java](https://marketplace.visualstudio.com/items?itemName=vscjava.vscode-java-debug) extension. (if not already installed.)

```

Comment:
Remove full stop after 'extension' and move final full stop outside parentheses thus:

`... extension (if not already installed).`